demos/background_subtraction_demo/cpp_gapi/src/custom_kernels.cpp

```diff
@@ -169,7 +169,7 @@ custom::MaskRCNNBGReplacer::MaskRCNNBGReplacer(const std::string& model_path) :
     }
 }
 
-cv::GMat custom::MaskRCNNBGReplacer::replace(cv::GMat in, const cv::Size& in_size, cv::GMat background) {
+cv::GMat custom::MaskRCNNBGReplacer::replace(cv::GFrame in, cv::GMat bgr, const cv::Size& in_size, cv::GMat background) {
```

Comment:
why do you need BGR here as an explicit parameter?

Can `BGR()` be called right here, in this function?

In the BGR GMat used somewhere else in the pipeline?

(If yes, we need to consider auto-optimizing this)

Reply:
BGR GMat might be used as a background in case `blur` option.

Reply:
Could you elaborate What do you mean by `auto-optimizing`